src/notebook/ipython_profiles/profile_narrative/kbase_templates/static/kbase/js/widgets/narrative_core/kbaseNarrativeMethodCell.js

```diff
@@ -10,7 +10,11 @@
  */
 
 (function( $, undefined ) {
-  require(['jquery', 'kbwidget'], function($) {
+  require(['jquery', 
+           'handlebars', 
+           'kbwidget', 
+           'kbaseAuthenticatedWidget',
+           'kbaseNarrativeCellMenu'], function($, Handlebars) {
```

Comment:
Ok, here is the App Cell dependency update:
https://github.com/kbase/narrative/pull/349